tests/test_heap.py

```diff
@@ -93,16 +94,16 @@ def setup_heap(start_binary, bug_no):
 
 def test_try_free_invalid_overflow(start_binary):
     chunks = setup_heap(start_binary, 1)
-    
+
     result = gdb.execute('try_free {}'.format(hex(chunks['a'])), to_string=True)
     assert 'free(): invalid pointer -> &chunk + chunk->size > max memory' in result
     os.remove(OUTPUT_FILE)
 
 
-# def test_try_free_invalid_misaligned(start_binary):
+def test_try_free_invalid_misaligned(start_binary):
```

Comment:
NEEDS ATTENTION: I assumed that it was commented as a mistake because at the end the test `test_try_free_invalid_overflow` execute its test and also the `test_try_free_invalid_misaligned` because the last one was only commented the `def` but not all the test. If it was intentional, please let me know and I'll restore it.